lib/utils/fs/writeFile.test.js

```diff
@@ -16,7 +16,7 @@ describe('#writeFile()', function () {
   this.timeout(0);
   beforeEach(() => {
     serverless = new Serverless();
-    serverless.init();
+    return serverless.init();
```

Comment:
Shouldn't we move this to the beginning of the `beforeEach` block?

Reply:
Here similarly, at beginning of this `beforeEach` block, there's no `serverless` instance